src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubListener.cs

```diff
@@ -12,180 +12,204 @@
 
 namespace Microsoft.Azure.WebJobs.ServiceBus
 {
-    // Created from the EventHubTrigger attribute to listen on the EventHub. 
```

Comment:
I reworked this entire file quite a bit to make it easier to understand and also to enable me to improve the test coverage.